<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CityBox</class>
 <widget class="QWidget" name="CityBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(0,0,0);
background-color: rgba(128, 128, 128, 20);</string>
  </property>
  <widget class="QPushButton" name="cityBtn">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-9</y>
     <width>240</width>
     <height>210</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{   
    border-top-left-radius: 15px;               /*圆角*/
	border-bottom-left-radius: 15px;  
	border-top-right-radius: 15px;  
	border-bottom-right-radius: 15px;  
	background-color: rgba(128, 128, 128, 20);
}

/* 鼠标悬停 */
QPushButton:hover{
	background-color: rgba(0, 0, 0,20);
}

QPushButton:checked{
	
	background-color: rgb(255, 180, 200);
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
